ipam/store.go

```diff
@@ -28,11 +28,11 @@ import (
 // that is to get an IP address.
 type Endpoint struct {
 	Ip           string         `json:"ip,omitempty"`
```

Comment:
In Host Message, we have "Id". Here we have "ID". I know this is not part of this PR, but I'm just pointing it out. Would be nice to have this more standardized.